.golangci.example.yml

```diff
@@ -559,6 +559,20 @@ linters-settings:
       - shadow
     disable-all: false
 
+  grouper:
+    # const
+    const-require-single-const: false
+    const-require-grouping: false
+    # import
+    import-require-single-import: false
+    import-require-grouping: false
+    # type
+    type-require-single-type: false
+    type-require-grouping: false
+    # var
+    var-require-single-var: false
+    var-require-grouping: false
```

Comment:
Can you improve the documentation?

- add an explanation for each option
- add the default value in comments for each option
- add a value for each option that is not the default value